Integrations/Whois/Whois.yml

```diff
@@ -28,6 +28,210 @@ script:
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.name
+      description: The name of tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.state
+      description: The state of tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.email
+      description: The email address of the tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.organization
+      description: The organization of the tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.postalcode
+      description: The postal code of the tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.street
+      description: The street of the tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.phone
+      description: The phone number of the tech administrator.
+      type: string
+    - contextPath: Domain.Whois.TechAdmin.city
+      description: The city of the tech administrator.
+      type: string
+    - contextPath: Domain.Whois.Registrant.country
+      description: The country of the registrant.
+      type: string
+    - contextPath: Domain.Whois.Registrant.state
+      description: The state of the registrant.
+      type: string
+    - contextPath: Domain.Whois.Registrant.organization
+      description: The organization of the registrant.
+      type: string
+    - contextPath: Domain.Whois.Registrant.postalcode
+      description: The postal code of the registrant.
+      type: string
+    - contextPath: Domain.Whois.Registrant.street
+      description: The street of the registrant.
+      type: string
+    - contextPath: Domain.Whois.Registrant.phone
+      description: The phone number of the registrant.
+      type: string
+    - contextPath: Domain.Whois.Registrant.city
+      description: The city of the registrant.
+      type: string
+    - contextPath: Domain.Whois.ID
+      description: The ID of the domain.
+      type: string
+    - contextPath: Domain.Whois.QueryStatus
+      description: The result of the command ("Success" or "Failed").
+      type: string
+  - arguments:
+    - default: true
+      description: The domain to enrich.
+      isArray: true
+      name: domain
+      required: true
+      secret: false
+    deprecated: false
+    description: Provides data enrichment for domains.
+    execution: false
+    name: domain
+    outputs:
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: string
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: string
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: string
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: string
+    - contextPath: Domain.Name
+      description: The domain name.
+      type: string
```

Comment:
@kirbles19 Can you please check here? It is copypasta from the other command since they are identical but just wanted your input.